x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -550,7 +613,7 @@ export const sendAlertToTimelineAction = async ({
       });
     }
   } else if (isThresholdRule(ecsData)) {
-    createThresholdTimeline(ecsData, createTimeline, noteContent, {});
+    return createThresholdTimeline(ecsData, createTimeline, noteContent, {});
```

Comment:
Is an `await` required in this context, similar to the  `return await createThresholdTimeline(ecsData, createTimeline, noteContent, {` [above](https://github.com/elastic/kibana/pull/124331/files#diff-972b23879f146e612234cef06ac25c143709767be35c69d621861d7b50d8ce24R558)?

Reply:
I think it's the other way actually, the await is redundant and not needed above. Not sure why the linter did not catch this one, it catches the other